### PR TITLE
PID: extend QA, add parametrizations for the TOF

### PR DESCRIPTION
**Pull request by njacazio**
- Add TOF calibration for ALICE3
- Add handler for Alice 3 param
- Update RICH QA task
- Add HMPID QA task
- Add task for FAST PID response
- Add beta in TOF QA task
- Remove fast and qa tasks
- Extend TOF task with Fast PID and more QA
- Add fast TPC response
- Add fast implementation in PID response of TPC and TOF
- Update TOF task for QA
- Update TPC param handler
- Add number of photons detected to the HMPID table
- Delegating track selection in QA to dedicated task
- Add ALICE3 track extension task
- Add ALICE3 task for multiplicity
- Add separate task for the TOF beta
- Update RICH table to build indices, use indeces in RICH QA task

Update PID param. handlers
- Writing TOF reso parameters and saving them to the CCDB
- Add ALICE3 PID TOF response

PID tasks:
- Add QA tasks to split task
- Add expected resolution and for the event time resolution plot to the QA of ALICE3 TOF PID
- Add cut on minimum TOF multiplicity for QA

### CODE REVIEW

File: Analysis/ALICE3/src/alice3-trackselection.cxx [hunk 1]
```diff
@@ -29,18 +28,49 @@ using namespace o2::framework::expressions;
...
+      if (track.eta() < MinEta || track.eta() > MaxEta) {
+        histos.fill(HIST("selection"), 2);
+        sel = false;
+      }
+      if (sel) {
+        histos.fill(HIST("dcaXY/selected"), track.dcaXY());
+        histos.fill(HIST("eta/selected"), track.eta());
+      }
+
+      filterTable(sel, sel);
```
Comment (iarsene) [inline Analysis/ALICE3/src/alice3-trackselection.cxx:73]: This table is filled twice with the same quantity. Is it a placeholder or a mistake?

**Comment by iarsene** [comment]
Hi @njacazio, the PR is quite large, is there someone else that should have a look into the affected files?

File: Analysis/ALICE3/src/alice3-trackselection.cxx (hunk 1 above)
Comment (njacazio) [reply Analysis/ALICE3/src/alice3-trackselection.cxx:73]: Hi, it's a placeholder

**Comment by njacazio** [comment]
Hi @iarsene, yes indeed it's quite large, my apologies. Specifically there is no one to look at this but I could try to split it into several sub PRs, the thing is that it is all referred to the same topic and it's quite interconnected but it's feasible.

**Review by iarsene — DISMISSED** [review]

**Comment by njacazio** [comment]
@iarsene unfortunately I missed an `override` this should be fix the macos compilation

**Comment by ktf** [comment]
I think this introduced:

```
/mnt/mesos/sandbox/sandbox/o2-fullci/sw/SOURCES/O2/5900/0/Analysis/DataModel/include/AnalysisDataModel/PID/ParamBase.h:74:16: error: 'virtual' is redundant since the function is already declared 'override' [modernize-use-override]
/mnt/mesos/sandbox/sandbox/o2-fullci/sw/SOURCES/O2/5900/0/Analysis/DataModel/include/AnalysisDataModel/PID/ParamBase.h:126:16: error: 'virtual' is redundant since the function is already declared 'override' [modernize-use-override]
/mnt/mesos/sandbox/sandbox/o2-fullci/sw/SOURCES/O2/5900/0/Analysis/DataModel/include/AnalysisDataModel/PID/ParamBase.h:74:16: error: 'virtual' is redundant since the function is already declared 'override' [modernize-use-override]
/mnt/mesos/sandbox/sandbox/o2-fullci/sw/SOURCES/O2/5900/0/Analysis/DataModel/include/AnalysisDataModel/PID/ParamBase.h:126:16: error: 'virtual' is redundant since the function is already declared 'override' [modernize-use-override]
/mnt/mesos/sandbox/sandbox/o2-fullci/sw/SOURCES/O2/5900/0/Analysis/DataModel/include/AnalysisDataModel/PID/ParamBase.h:74:16: error: 'virtual' is redundant since the function is already declared 'override' [modernize-use-override]
/mnt/mesos/sandbox/sandbox/o2-fullci/sw/SOURCES/O2/5900/0/Analysis/DataModel/include/AnalysisDataModel/PID/ParamBase.h:126:16: error: 'virtual' is redundant since the function is already declared 'override' [modernize-use-override]
/mnt/mesos/sandbox/sandbox/o2-fullci/sw/SOURCES/O2/5900/0/Analysis/DataModel/include/AnalysisDataModel/PID/ParamBase.h:74:16: error: 'virtual' is redundant since the function is already declared 'override' [modernize-use-override]
/mnt/mesos/sandbox/sandbox/o2-fullci/sw/SOURCES/O2/5900/0/Analysis/DataModel/include/AnalysisDataModel/PID/ParamBase.h:126:16: error: 'virtual' is redundant since the function is already declared 'override' [modernize-use-override]
/mnt/mesos/sandbox/sandbox/o2-fullci/sw/SOURCES/O2/5900/0/Analysis/DataModel/include/AnalysisDataModel/PID/ParamBase.h:74:16: error: 'virtual' is redundant since the function is already declared 'override' [modernize-use-override]
/mnt/mesos/sandbox/sandbox/o2-fullci/sw/SOURCES/O2/5900/0/Analysis/DataModel/include/AnalysisDataModel/PID/ParamBase.h:126:16: error: 'virtual' is redundant since the function is already declared 'override' [modernize-use-override]
/mnt/mesos/sandbox/sandbox/o2-fullci/sw/SOURCES/O2/5900/0/Analysis/DataModel/include/AnalysisDataModel/PID/ParamBase.h:74:16: error: 'virtual' is redundant since the function is already declared 'override' [modernize-use-override]
/mnt/mesos/sandbox/sandbox/o2-fullci/sw/SOURCES/O2/5900/0/Analysis/DataModel/include/AnalysisDataModel/PID/ParamBase.h:126:16: error: 'virtual' is redundant since the function is already declared 'override' [modernize-use-override]
/mnt/mesos/sandbox/sandbox/o2-fullci/sw/SOURCES/O2/5900/0/Analysis/DataModel/include/AnalysisDataModel/PID/ParamBase.h:74:16: error: 'virtual' is redundant since the function is already declared 'override' [modernize-use-override]
/mnt/mesos/sandbox/sandbox/o2-fullci/sw/SOURCES/O2/5900/0/Analysis/DataModel/include/AnalysisDataModel/PID/ParamBase.h:126:16: error: 'virtual' is redundant since the function is already declared 'override' [modernize-use-override]
```

can you please have a look?

**Comment by njacazio** [comment]
@ktf sure, I'm having a look

**Pull request merged**